examples/go/tagging/handler/handler_test.go

```diff
@@ -0,0 +1,128 @@
+package function
+
+import (
+	"io/ioutil"
+	"testing"
+)
+
+const passMark = "\u2713"
+const failMark = "\u2717"
+
+// TestLoadTomlCfg ensures only valid vcconfig.toml files are loaded and processed.
+func TestLoadTomlCfg(t *testing.T) {
+	var tomls = []struct {
+		testDesc   string
+		configPath string
+		vcServer   string
+		vcUser     string
+		vcPassword string
+		tagUrn     string
+		tagAction  string
+	}{
+		{
+			"Toml file should load correctly",
+			"testdata/vcconfig.toml",
+			"veba.local.corp",
+			"admin@vsphere.local",
+			"password1234",
+			"urn:vmomi:InventoryServiceTag:11f16f36-f5c4-4c29-b7d3-d9c7d12babe6:GLOBAL",
+			"attach",
+		},
+		{
+			"Toml file should load, even if it has more information than needed",
+			"testdata/vcconfig2.toml",
+			"veba.local.corp",
+			"admin@vsphere.local",
+			"password1234",
+			"urn:vmomi:InventoryServiceTag:11f16f36-f5c4-4c29-b7d3-d9c7d12babe6:GLOBAL",
+			"detach",
+		},
+	}
+
+	for _, v := range tomls {
```

Comment:
Convention for table-driven tests is usually []tests and tt as var

Reply:
Also table-driven tests usually use structs for want/got to simplify the []tests structure (readability). This lets you compare structs instead of many strings and simplifies the test structure.